src/ast-helpers.ts

```diff
@@ -11,3 +13,29 @@ export function getReactImportReference(ast): t.ImportDeclaration {
 export function isExportedDeclaration(ast) {
   return t.isExportNamedDeclaration(ast) || t.isExportDefaultDeclaration(ast);
 }
+
+export function findPathInContext(ast, identifierName) {
+  let foundPath = null;
+  const visitor = {
+    Identifier(path) {
+      if (!foundPath && path.node.name === identifierName) {
+        foundPath = path;
+      }
+    }
+  };
+
+  traverse(ast, visitor);
+  return foundPath;
+}
+
+export function pathContains(path, start, end) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'end' to have a typedef](https://app.codacy.com/gh/wix/vscode-glean/pullRequest?prid=5535229)